### PR TITLE
[docs] Add code example for formatted input fields (typeText)

### LINKED ISSUE #5573: Add test code examples how to set value for datetime, range and color inputs
Connected with https://github.com/DevExpress/testcafe/issues/5571.

https://devexpress.github.io/testcafe/documentation/reference/test-api/testcontroller/typetext.html#typing-into-datetime-color-and-range-inputs

### PR DESCRIPTION
**Pull request by DIRECTcut**
fixes #5573

### CODE REVIEW

**Review by miherlosev — APPROVED** [review]

**Review by VasilyStrelyaev — APPROVED** [review]

File: docs/articles/documentation/reference/test-api/testcontroller/typetext.md [hunk 1]
```diff
@@ -13,11 +13,11 @@ Types the specified text into an input element.
...
+Parameter                   | Type                                              | Description
+--------------------------- | ------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------
+`selector`                  | Function &#124; String &#124; Selector &#124; Snapshot &#124; Promise | Identifies the webpage element that receives input focus. See [Select Target Elements](#select-target-elements).
+`text`                      | String                                            | The text to be typed into the specified webpage element.
+`options`&#160;*(optional)* | Object                                            | A set of options with additional parameters for the action. See [Options](#options). If this parameter is omitted, TestCafe sets the cursor to the end of the text before typing. This preserves the text that is already in the input box.
```
Comment (dirk-pieterse) [inline docs/articles/documentation/reference/test-api/testcontroller/typetext.md:20]:
```suggestion
`options`&#160;*(optional)* | Object                                            | A set of options with additional parameters for the action. See [Options](#options). If this parameter is omitted, TestCafe moves the cursor to the end of the text before typing. This preserves the text that is already in the input box.
```

File: docs/articles/documentation/reference/test-api/testcontroller/typetext.md [hunk 2]
```diff
@@ -42,9 +42,9 @@ test('Type and Replace', async t => {
 });
 ```
 
-## Typing Into DateTime, Color and Range Inputs
+## Typing Into DateTime, Color, and Range Inputs
 
-There are certain types of HTML5 inputs, like `DateTime`, `Color` or `Range`, that require entering values in a specific format.
+Some HTML5 inputs, like `DateTime`, `Color` or `Range`, require specifically formatted values.
```
Comment (dirk-pieterse) [inline docs/articles/documentation/reference/test-api/testcontroller/typetext.md:47]: What are "specifically formatted values"? Can you explain?

File: docs/articles/documentation/reference/test-api/testcontroller/typetext.md [hunk 3]
```diff
@@ -55,9 +55,30 @@ Week       | `yyyy-Www`         | `'2017-W03'`
 Month      | `yyyy-mm`          | `'2017-08'`
 DateTime   | `yyyy-mm-ddThh:mm` | `'2017-11-03T05:00'`
 Time       | `hh:mm`            | `'15:30'`
-Color      | `#rrggbb`          | `'#003000'`
+Color      | `#rrggbb` (hex)    | `'#FF8040'`
 Range      | `n`                | `'45'`
 
+The following example uses `t.typeText` to fill `color`, `datetime-local` and `range` input fields.
```
Comment (dirk-pieterse) [inline docs/articles/documentation/reference/test-api/testcontroller/typetext.md:61]:
```suggestion
The following example uses `t.typeText` to fill `color`, `datetime-local` and `range` input fields:
```

**Review by dirk-pieterse — APPROVED** [review]

File: docs/articles/documentation/reference/test-api/testcontroller/typetext.md (hunk 2 above)
Comment (DIRECTcut) [inline docs/articles/documentation/reference/test-api/testcontroller/typetext.md:47]:
```suggestion
Some HTML5 inputs, like `DateTime`, `Color` or `Range`, require formatted values.
```

**Comment by DIRECTcut** [comment]
@testcafe-build-bot \retest

**Pull request merged**